### PR TITLE
Csv export

### PR DESCRIPTION
**Pull request by spearna**
A basic implementation using a URL path to generate a CSV roster of everyone who's entered their info.
Closes issue #13

### CODE REVIEW

File: app/lib/routes.rb [hunk 1]
```diff
@@ -58,6 +58,21 @@ def form(request, view, form)
...
+
+  get '/export' do
+    # Feature to locally export and save a roster.
+    # Grab all Profile entries, build CSV, export to local machine via browser
+    @profiles = Profile.all
```
Comment (tessellator) [inline app/lib/routes.rb]: It is better to use a local variable and not a member variable (e.g., prefer `profiles` to `@profiles`) since it is only used internally to this method.

Comment (spearna) [inline app/lib/routes.rb]: I don't fully understand the implications, so I'll defer to your recommendation.

Comment (tessellator) [inline app/lib/routes.rb]: I don't know why the behavior would have changed, but if it works go ahead and push it. We'll write some tests to work out the edge cases afterward (a little backwards, but we're being a bit "cowboy" for this project right now anyway).

**Comment by tessellator** [comment]
As for the implications...it's best to keep the scope of a variable as local as possible. Smaller scopes are generally less complex and easier to manage and change. In this case, you promoted `profiles` to be a member variable on whatever class is generated by `Pakyow::App.routes`. Since there is nowhere else in the class that needs this data, you are adding extra data in that larger scope (remember that a local function scope is "smaller" than a class scope).

Keeping variables as locally scoped as possible supports the [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single_responsibility_principle) (aka SRP). SRP is a major influence, at least to me, on how to build simple software. By promoting `profiles` to a class member, you are beginning to muddy the purpose of the class generated by `Pakyow::App.routes`, which is just to set up routes (not set up routes _and_ hold profile information).
